operators/operator_sdk/osdk-about.adoc

```diff
@@ -25,9 +25,39 @@ The Operator SDK is a framework that uses the link:https://github.com/kubernetes
 
 Operator authors with cluster administrator access to a Kubernetes-based cluster (such as {product-title}) can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
 
-[NOTE]
+[id="osdk-compatibility-matrix"]
+== Operator SDK compatibility matrix
+
+
+[WARNING]
+====
+The compatibility asserts the compatibility of the operator SDK itself, it does not take into consideration the API calls done by your code.
+These must be asserted by the developer of the operator.
+====
+
+**Helm-based and Ansible-based operators compatibility**
+
+Operators based in Helm and Ansible must follow a predefined project layout. There are breaking changes between Operator SDK 0.19 and 1.3
+
+[options="header"]
+|===============================================================================================================
+|Operator SDK version | Builder image version | Compatible OpenShift version | Compatible Kubernetes API version 
+| 0.19                | 4.6                   | 4.6 and 4.7                  | 1.18, 1.19
+| 1.3                 | 4.7                   | 4.6 - 4.8                    | 1.18 - 1.20
+|===============================================================================================================
+
+**Go operators compatibility**
+
+[options="header"]
+|===================================================================================================================
+|Operator SDK version | Go version | K8s version | Compatible OpenShift versions | Compatible Kubernetes API version 
+| 0.19                | 1.13       | 1.18        | 4.6 and 4.7                   | 1.18, 1.19
+| 1.3                 | 1.15       | 1.19        | 4.6 - 4.8                     | 1.18 - 1.20
```

Comment:
```suggestion
| 1.3                 | 1.15       | 1.19        | 4.6 - 4.8                     | 1.18 - 1.20*

* you would have to update your k8s client to 1.20 in your go.mod to access 1.20 APIs. 
```

Not sure how we handle footnotes like that.

Reply:
Maybe a more accurate statement would be doing it like  [compatibility matrix](https://github.com/kubernetes/client-go/tree/master#compatibility-matrix)
| 4.6⁻, 4.7, 4.8⁺      | 1.18⁻, 1.19, 1.20⁺


* + client-go has features or API objects that may not be present in the Kubernetes cluster, either due to that client-go has additional new API, or that the server has removed old API. However, everything they have in common (i.e., most APIs) will work. Please note that alpha APIs may vanish or change significantly in a single release.
* - The Kubernetes cluster has features the client-go library can't use, either due to the server has additional new API, or that client-go has removed old API. However, everything they share in common (i.e., most APIs) will work.